ChangeLog.md

```diff
@@ -1,5 +1,153 @@
 # Changelog for haskell-language-server
 
+## 1.3.0
+
+2021 July release of HLS arrives! This release includes binaries for GHC 9.0.1
+and some new interesting features. Here is the brief summary of changes:
+
+- Binaries for GHC 9.0.1 are added by @anka-213.
+- Call hierarchy plugin is added, contributed by @July541.
+  ![hierarchy](https://user-images.githubusercontent.com/12473268/127550041-094151a6-be7b-484a-bb82-c61f326ca503.gif)
+- Now completions work with definitions from non-imported modules, thanks to @pepeiborra.
+  ![completion](https://user-images.githubusercontent.com/12473268/127543694-718ae043-38f2-4fb0-be71-317f5f93b443.gif)
+- Eval plugin
+  - The plugin supports GHC 9.0.1, thanks to @berberman.
+  - `:info` command is added by @akrmn.
+  - The plugin uses the same default language as GHCi with @fmehta's patch.
+- Wingman, where most changes owing to @isovector
+  - An option for disabling proof state styling is added.
```

Comment:
```suggestion
  - Wingman no longer changes the fixity of function definitions.
  - Wingman now gives unique names to the holes it generates.
  - Significant improvements in Wingman's ability to reason about polymorphic and GADT types.
  - Wingman no longer suggests homomorphic destructs when the codomain is larger than the domain.
  - "Complete case constructors" support for empty lambda cases
  - Wingman will now give a warning if it ran out of gas during "attempt to fill hole"
  - Metaprogramming improvements, including symbolic-name support, and the `pointwise` combinator.
  - Fixed a bug when attempting to introduce too many variables in an `intros` metaprogram.
```

Reply:
May I ask why you want to remove the line for proof styling option?

Besides that,

> - Fixed a bug when attempting to introduce too many variables in an `intros` metaprogram.

should these be highlighted?

Reply:
> May I ask why you want to remove the line for proof state styling option?

it's the least interesting thing in the change, imo. does anyone even know what a proof state is?